### DISCUSSION TITLE
Beginner in NextJS - Problem advancing on the tutorial

### DISCUSSION THREAD

**Discussion by Rafaelius** [vercel/next.js#12720] [Help]
Hi, i'm new at NextJS so i'm taking the simple Blog tutorial.

As i reach this step: https://nextjs.org/learn/basics/assets-metadata-css/layout-component
i'm facing this issue here:

./pages/posts/first-post.js
Module not found: Can't resolve '../../components/layout' (...)

Already tried different approaches on how to write the path, but nothing solved the problem and i can't move on the tutorial.

Could you help?

Thanks!

**Comment by pavittarx** [comment]
* Does the module exists at "../../components/layout"; Check the path relative from the file it is imported in.

If the module exists try changing it to "./../../components/layout";

I experienced a similar problem a few days ago, but found that prefixing "./" to the import url solved this. FYI, "./" references the current directory.

**Reply by Rafaelius** [reply]
Yes, i tried those different paths, but the problem remains.

Could it be somenthing about configuration in node or something like it?
I'm at Win10 by the way

node 12.16
npm 6.14

**Reply by rafaelalmeidatk** [reply]
Can you upload your project somewhere (like GitHub) with the reproduction so we can give it a closer look?
